src/WindowManager.vala

```diff
@@ -410,10 +410,12 @@ namespace Gala {
         private void launch_action (string action_key) {
             try {
                 var action = behavior_settings.get_string (action_key);
-                if (action != null && action != "") {
```

Comment:
Is `action != null` the same as `action != ""`?

Reply:
No, I removed it because if `action` is empty it would produce a warning instead of failing silently.